modules/standard/Sets.chpl

```diff
@@ -0,0 +1,395 @@
+      the elements contained in another set.
+
+      :arg other: The set to initialize from.
+      :arg parSafe: If `true`, this set will use parallel safe operations.
+    */
+    proc init=(const ref other: set(?t), param parSafe=other.parSafe) {
+    }
+
+    pragma "no doc"
+    inline proc deinit() {
+    }
+
+    pragma "no doc"
+    inline proc _enter() {
+      if parSafe then
+        _lock$.lock();
+    }
+
+    pragma "no doc"
+    inline proc _leave() {
+      if parSafe then
+        _lock$.unlock();
+    }
+
+    /*
+      Add a copy of the element `x` to this set. Does nothing if this set
+      already contains an element equal to the value of `x`.
+
+      :arg x: The element to add to this set.
+    */
+    proc add(in x: eltType) {
+      return;
+    }
+
+    /*
+      Returns `true` if the given element is a member of this set, and `false`
+      otherwise.
+
+      :arg x: The element to test for membership.
+      :return: Whether or not the given element is a member of this set.
+    */
+    proc const contains(const ref x: eltType): bool {
+      return false;
+    }
+
+    /*
+      Returns `true` if this set shares no elements in common with the set
+      `other`, and `false` otherwise.
+
+      :arg other: The set to compare against.
+      :return: Whether or not this set and `other` are disjoint.
+    */
+    proc const isDisjoint(const ref other: set(eltType, ?)): bool {
+      return false;
+    }
+
+    /*
+      Returns `true` if this set and `other` have at least one element in
+      common.
+
+      :arg other: The set to compare against.
+      :return: Whether or not this set and `other` intersect.
+    */
+    proc const isIntersecting(const ref other: set(eltType, ?)): bool {
+      return false;
+    }
+
+    /*
+      Remove the item from this set with a value equal to `x`.
+
+      .. warning::
+
+        Removing an element from this set may invalidate existing references
+        to the elements contained in this set.
+    
+      :arg x: The element to remove.
+
+      :throws IllegalArgumentError: If the list contains no such element.
+    */
+    proc remove(const ref x: eltType): eltType throws {
+      return;
+    }
+
+    /*
+      Remove the item `x` if it is present in this set, but do nothing if it
+      is not.
+
+      .. warning::
+
+        Discarding an element from this set may invalidate existing references
+        to the elements contained in this set.
+
+      :arg x: The element to discard.
+    */
+    proc discard(const ref x: eltType) {
+      return;
+    }
+
+    /*
+      Clear the contents of this set.
+
+      .. warning::
+
+        Clearing the contents of this set will invalidate all existing
+        references to the elements contained in this set.
+    */
+    proc clear() {
+      return;
+    }
+
+    /*
+      Iterate over the elements of this set.
+
+      :yields: A reference to one of the elements contained in this set.
```

Comment:
Other iterator docstrings in `modules/` describe the yields overall, not a single yielded instance.  e.g., 
```
modules/packages/DistributedIters.chpl:  :yields: Indices in the range ``c``.
```

Does it yield them in the same sorted order as `toArray()`?